packages/bcer-api/app/src/location/enums/location-status.enum.ts

```diff
@@ -0,0 +1,5 @@
+export enum LocationStatus {
+    ACTIVE = 'active',
+    CLOSED = 'closed',
+    DELETED = 'deleted',
+}
```

Comment:
btw, Enum is better to use PascalCase.

![image](https://user-images.githubusercontent.com/46575401/137948620-2405e412-6beb-4a25-89b4-670bebc5b6d0.png)

Reply:
`Active`, `Closed`...